amethyst_controls/src/resources.rs

```diff
@@ -1,12 +0,0 @@
-/// Struct which holds information about whether the window is focused.
-/// Written to by MouseFocusUpdateSystem
-#[derive(Default)]
-pub struct WindowFocus {
```

Comment:
This is useful for other use-cases, please don't remove it.

Reply:
Agreed.